### ISSUE TITLE
Tests: re-enable node 4 and 5 tests

### ISSUE THREAD

**Issue by ljharb** [import-js/eslint-plugin-import#2053]
They are likely legitimately failing right now, but we need to unblock a release.

**Comment by soryy708** [comment]
Node 4 is already enabled:

- https://github.com/import-js/eslint-plugin-import/blob/fd1ae63492a554ded6c5cc7fc262034804819e5e/.github/workflows/native-wsl.yml#L19
- https://github.com/import-js/eslint-plugin-import/blob/fd1ae63492a554ded6c5cc7fc262034804819e5e/.github/workflows/node-4%2B.yml#L127-L136

Node 5 is also already enabled:

- https://github.com/import-js/eslint-plugin-import/blob/fd1ae63492a554ded6c5cc7fc262034804819e5e/.github/workflows/node-4%2B.yml#L117-L126

Therefore looks to me like this was already completed

**Comment by ljharb** [comment]
node 4 is enabled on windows, but the other things you listed are specific exclude combinations - the tests still aren't running.

**Comment by MichaelDeBoey** [comment]
@ljharb I guess these tests will be obsolete once #3091 is merged?

**Comment by ljharb** [comment]
@MichaelDeBoey yep

**Comment by germancasares** [comment]
I don't know enough of the project to help, but it looks like the progress on that PR has stagnated or at least it looks like it is not a priority.

Merging that one will unblock things like bumping the minimatch library to support more patterns.

Ref: https://github.com/import-js/eslint-plugin-import/issues/2900#issuecomment-1859477364
